gserver/src/steps.handler.ts

```diff
@@ -206,8 +206,8 @@ export default class StepsHandler {
         //https://docs.cucumber.io/cucumber/cucumber-expressions/
         step = step.replace(/{float}/g, '-?\\d*\\.?\\d+');
         step = step.replace(/{int}/g, '-?\\d+');
-        step = step.replace(/{stringInDoubleQuotes}/g, '"[^"]+"');
```

Comment:
Not sure that we should remove `stringInDoubleQuotes`. It was supported some time ago, so somebody could use it.

Reply:
I will re-add for backwards compatibility. we should consider adding a note in the release notes that this will be removed in the future.